src/js/utils/dom.js

```diff
@@ -209,3 +209,11 @@ export function nextSibling(element) {
 export function previousSibling(element) {
     return element.previousElementSibling;
 }
+
+export function open(link, target, additionalOptions = {}) {
```

Comment:
this is super pedantic but `openLink` might be more descriptive here - less potential for variable collisions and whatnot too